Add note to bug-report template (#4876)
dotnet/roslyn-analyzers: .github/ISSUE_TEMPLATE/bug-report.md

**Version**: [SDK 5.0.100](https://dotnet.microsoft.com/download/dotnet/5.0)

<!--

> Note: Updates to NuGet package are more frequent than SDK, which means the NuGet package has fixes that didn't yet made into the SDK.

Reviewer: ```suggestion
Note: Updates to NuGet package are more frequent than SDK, which means the NuGet package has fixes that haven't yet made it into the SDK.
```

Reviewer: Indent is to have a uniform look with the other note later in the file.

> So it's preferred you check whether the bug you encountered has been fixed in the latest NuGet package or not.

Reviewer: ```suggestion
      Please check whether you can reproduce the bug you encountered in the latest NuGet package.
```
